Adding Initial Authentication Code

mtolrom opened the pull request:
Adding Initial Authentication Code

mviencek commented:
i forgot to say that removing     <link rel="stylesheet" href="views/styles/styles.css"> broke several things

mviencek commented:
link rel="stylesheet" href="views/styles/styles.css"

mtolrom commented:
I added that config file only for testing. will remove that.
I removed the CSS to let UI people fix it. It's shifting menu items an in incorrect display.
Insert queries will not be used. Back-End team should replace that with store procedures.

mviencek commented:
no worries  about the sql.  for the rest of it though please remove config.php and remap references to configuration.php also add nbproject/project.properties and nbproject/project.xml to gitignore.

mcontado commented on views/static/config.php:
```diff
@@ -0,0 +1,15 @@
+<?php
```
remove this file. we already have configuration.php

mcontado commented on views/static/login.php:
```diff
@@ -0,0 +1,36 @@
+<?php
```
remove this file, Mekone is working on this.

mcontado commented on views/static/logout.php:
```diff
@@ -0,0 +1,6 @@
+<?php
```
remove this file, Mekone is working on this.

mcontado commented on views/static/profile.php:
```diff
@@ -0,0 +1,30 @@
+<?php
```
remove this file, Mekone is working on this.

mcontado commented on views/static/registration.php:
```diff
@@ -1,3 +1,42 @@
+<?php
```
please checkout the latest registration page from the Development. try doing: ```git checkout origin/Development views/static/registration.php```

mcontado commented on views/static/search_events.php:
```diff
@@ -1,10 +1,12 @@
- * Date: 5/8/2018
- * Time: 3:55 PM
- */
+
```
remove this file, Mekone is working on this.

mcontado commented on views/templates/header.php:
```diff
@@ -1,14 +1,25 @@
-        </ul>
-    </nav>
-</div>
+<nav class="navbar navbar-expand-sm bg-dark navbar-dark">
```
remove this file, Mekone is working on this.

mtolrom commented:
Changes have been pushed.

mviencek commented:
cool thank you.  i noticed that there are hard coded values in this query
  $query = "INSERT INTO Users(email, password, firstName, lastName, phoneNumber, notificationState, UserTypeId) VALUES ('$emailAddress', '$password', '$firstName', '$lastName', '$phoneNumber', 1, 1)";

was that intentional?
also could you also add the link back to the styles page.  they are aware that there is problems with the file.  they are going to fix them.

mviencek commented on nbproject/project.xml:
```diff
@@ -0,0 +1,9 @@
+            <name>PhpProject1</name>
+        </data>
+    </configuration>
+</project>
```
your nbproject stuff is still coming through

mviencek commented on nbproject/project.properties:
```diff
@@ -0,0 +1,7 @@
+php.version=PHP_70
+source.encoding=UTF-8
+src.dir=.
+tags.asp=false
```
your nbproject stuff is still coming through

mviencek commented:
Can you close your pull request and resubmit it under a different name?  It's causing confusion with Development.  People aren't sure which one to merge with.  Thank you
